tests/libcxx/expected_results.txt

```diff
@@ -792,7 +724,7 @@ std/localization/locale.categories/category.ctype/locale.codecvt/locale.codecvt.
 std/localization/locale.categories/category.ctype/locale.codecvt/locale.codecvt.members/char32_t_max_length.pass.cpp FAIL
 
 # Likely STL bug in <chrono>: "result type of conditional expression is ambiguous"
-std/utilities/time/time.duration/time.duration.alg/abs.pass.cpp FAIL
+std/time/time.duration/time.duration.alg/abs.pass.cpp FAIL

```

Comment:
This might be LWG-3741 "`std::abs(duration)` is ill-formed with non-reduced periods".

Reply:
`# GH-2839 <chrono>: abs(duration) requires a duration with reduced period`

...which will eventually be fixed by #2988.